### ISSUE TITLE
Enhancement Request: Add own links to Share

### ISSUE THREAD

**Issue by ShadowEO** [openam/calibrephp#9]
I have an enhancement request and a question about the book view component.

My enhancement request is that I'd like to add an extra action to either the share or download buttons, the reasoning is below in my question as well.

I'm working on a separate webapp that uses Google Apps to push a ePub to Google Play Books (or at least I hope it will.)

However I'm not sure where I can add the button at. I was going to create a quick hack to the display function for downloads however I'm not sure how one would go about doing so (I'm looking through the code right now for places I can stick this link in.)

Basically what I believe would work would be a dirty hack, my application receives the book number from CalibrePHP and then file_get_contents() the download link, it then uploads it to the user's GDrive account and from there to Google Play Books. Would I be able to use file_get_contents on the links generated to do so?
